### DISCUSSION TITLE
[com_fields] List all Categories - no category fields output

### DISCUSSION THREAD

**Discussion by pepperstreet** [joomla/joomla-cms#19861] [2. Cold Features]
### Steps to reproduce the issue

Create custom fields for one or multiple **Categories**. 
Create some fields for any **automatic** display option. For instance AFTER TITLE. 
Create some fields for **manual** display through plugin syntax in category description. 
BTW, you have to memorize the field ID and insert the plugin syntax by hand. No Editor button!  
Create a menuItem of type "**List all categories**". Show descriptions.

### Expected result

Category fields should display on the listing, for each category.
Either through automatic-display, or plugin-syntax.

### Actual result

No fields display.

### System information (as much as possible)

Joomla 3.8.5

### Additional comments

Related issue: The editor button for fields modal is not displayed! 

Related views are: 
../components/com_content/views/categories/tmpl/default.php
../layouts/joomla/content/categories_default.php
../components/com_content/views/categories/tmpl/default_items.php
../components/com_content/views/categories/view.html.php

CC: @laoneo (mention as requested on a related facebook group comment)

**Comment by laoneo** [comment]
The events are missing completely. If this gets implemented, then probably not before 3.9 or 4.0.

**Comment by brianteeman** [comment]
Closed as a duplicate of #18457

**Comment by pepperstreet** [comment]
@laoneo 
Thanks for checking the issue and your short explanation. You've mentioned the VIEW.HTML on facebook, is this the file which should have the "triggers"? I am not that firm in API and coding, but a very long time user of Joomla. I just noticed the almost empty file and compared it with the usual blog view.html. A big difference.
BTW, the latter one seem to have dispatcher lines in the view layout **and** also in the view.html file?! Which confused me even more ;)

I have **updated** the initial post with a related issue:  
The fields **editor button** does not show up in TinyMCE toolbar?! So the plugin syntax has to be typed by hand.

**Comment by pepperstreet** [comment]
@brianteeman 
This is no duplicate of #18457 , which refers to a different menuItem type "Category list". 
My topic refers to the "List all Categories". Please, re-open it. 
TIA
